config/runtimes/cluster-scope/mlserver-0.x.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2021 IBM Corporation
```

Comment:
I think using kustomize we can avoid new yaml files. We can change the existing ones to be `ClusterServingRuntime`s and then for namespace scope install have an overlay which patches them like:

```yaml
patchesJSON6902:
- target:
    group: serving.kserve.io
    version: v1alpha1
    kind: ClusterServingRuntime
    name: ".*"
  patch: |-
    - op: replace
      path: /kind
      value: ServingRuntime
```